soc/ambiq/apollo4x/Kconfig.defconfig

```diff
@@ -6,4 +6,9 @@ if SOC_SERIES_APOLLO4X
 
 rsource "Kconfig.defconfig.apollo4*"
 
+# Need to enlarge the IDLE stack size because the power
+# management operations are executed in the idle task
+config IDLE_STACK_SIZE
+	default 2048 if PM
```

Comment:
This is quite the increase from the default 256, how was the value chosen?

Reply:
It is, yes. The IDLE_STACK_SIZE was chosen such that all of our current samples and tests would pass.